lib/workload/components/python-metadata-tools-layer/metadata_tools_layer/src/metadata_tools/utils/library_helpers.py

```diff
@@ -94,6 +94,28 @@ def get_library_workflow(library_id: str) -> Union[str | None]:
     return library.get("workflow")
 
 
+def get_library_project_owner(library_id: str) -> Union[str | None]:
+    """
+    Given a library id, collect the library project owner
+    :param library_id:
+    :return:
+    """
+    library = get_library_from_library_id(library_id)
+
+    return library.get("project_owner")
```

Comment:
On the casing end, I suppose this `get` will take value directly from the results JSON?  (From #502) the metadata now has a middleware to translate snake_case to pascalCase aligning with other microservices, so this might not get project_owner properly.

In terms of query params, the API may still support snake_case filters, but would recommend using the pascalCase if you can as this is what the Swagger documents it.

Reply:
I guess this is the same for library_id/Id and subject_id/Id too?

Reply:
Yes, the response will be in camelCase

Reply:
Yes, the response will be in camelCase